dead thoughts (starclan edition)

capillarys opened the pull request:
initially was only going to do newborns and medicine cats, but kind of went ham :P now have gotten new thoughts for a lot of dead cats (apps, deputy, med, kits, potats)!! praying nothing breaks </3

capillarys committed:
dead med cat thoughts

dunno if this works, let's try?

capillarys committed:
comma in med cat thought

commas are my enemy

capillarys committed:
entered some more thoughts

tysm ronfaure for your ideas! added some of em in

capillarys committed:
added a lot of dead starclan thoughts + changed id names

added pretty much all the thoughts from the ideas doc!
also renamed some IDs and moved things around (was told to move multi-group thoughts to general instead)
also added a LOT of thoughts, hope none of them will break the game T.T

capillarys committed:
+ more thoughts

okay turns out i missed a few (didn't save), here goes

capillarys committed:
an apostrophe

curly apostrophes my beloathed

scribblecrumb commented on resources/dicts/thoughts/dead/starclan/newborn.json:
```diff
@@ -1,22 +1,28 @@
+            "Wobbles around StarClan happily",
+            "Is drinking milk from a StarClan queen",
+            "Tries opening their eyes to see the 'stars' everyone is talking about",
+            "Is drinking milk from a StarClan queen",
```
this is a duplicate thought

scribblecrumb commented on resources/dicts/thoughts/dead/starclan/newborn.json:
```diff
@@ -27,5 +33,20 @@
+        "id": "gen_dead_newborn_to_dead_other",
+        "thoughts":[
+            "Is being watched by r_c",
+            "r_c is bathing them",
```
"is being bathed by r_c" fits the thought format better

capillarys replied:
sorry! i copied it over from the ideas doc and didn't proofread, my bad

capillarys committed:
scrimbly changes

wording, removed duplicate

capillarys commented:
wait lemme add some more thoughts for the roles that didn't have much/any

capillarys committed:
More Thoughts

i haven't done warrior, elder, leader yet, but i've fluffed out kit thoughts better, and i don't even know what i've added at this point. would prob want to reorganize the general part, mainly bc rn it's everything that has multiple "groups" of constraints...
